# Generated by Django 3.2.14 on 2022-08-06 10:30

from django.conf import settings
from django.db import migrations, models
import django.db.models.deletion


class Migration(migrations.Migration):

    dependencies = [
        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
        ('orders', '0006_alter_order_notes'),
    ]

    operations = [
        migrations.CreateModel(
            name='canceled_orders',
            fields=[
                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('reason_for_cancel', models.TextField(blank=True, choices=[('order  placed by mistake', 'order  placed by mistake'), ('price for the product has decreased', 'price for the product has decreased'), ('i have purchased the product elsewhere', 'i have purchased the product elsewhere'), ('i want to cance  due to the prodduct quality issues', 'i want to cance  due to the prodduct quality issues')])),
                ('order', models.ForeignKey(on_delete=django.db.models.deletion.DO_NOTHING, to='orders.order')),
                ('user', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL)),
            ],
        ),
    ]
